pkg/router/template/router.go

```diff
@@ -450,6 +451,8 @@ func (r *templateRouter) AddRoute(id string, route *routeapi.Route, host string)
 		}
 	}
 
+	config.RoutingKeyName = fmt.Sprintf("%x", md5.Sum([]byte(fmt.Sprintf("%s %s", config.TLSTermination, backendKey))))
```

Comment:
Yeah, so its just for uniqueness and it would probably become more readable if we just used a temp
variable to store the `key` and then hash it and set `config.RoutingKey` ala:
```  
key := fmt.Sprintf("%s %s", config.TLSTermination, backendKey)  
config.RoutingKeyName = fmt.Sprintf("%x", md5.Sum([]byte(key)))  
```